### PR TITLE
Add simulation time to FrameData for ASESimulation

### PR DESCRIPTION
**Pull request by hjstroud**
This PR adds the time elapsed during the simulation to the FrameData delivered to the user during an ASE simulation using the ASESimulation class. To do so requires a conversion from the [internal units of ASE](https://wiki.fysik.dtu.dk/ase/ase/units.html) to femtoseconds, and a subsequent conversion to picoseconds.

### CODE REVIEW

**Comment by Ragzouken** [comment]
We should probably add this to OMM too if it's not too difficult to include it in this request?

**Comment by hjstroud** [comment]
> We should probably add this to OMM too if it's not too difficult to include it in this request?

I believe OMM already delivers this information, this is to bring ASE in line with OMM

File: python-libraries/nanover-omni/src/nanover/omni/ase.py [hunk 1]
```diff
@@ -172,6 +176,9 @@ def advance_to_next_report(self):
         # generate the next frame
         frame_data = self.make_regular_frame()
 
+        # Add simulation time to the frame
+        frame_data.simulation_time = self.dynamics.get_time() * ASE_TIME_UNIT_TO_FS
```
[the remark this answers is not in the document]
Comment (hjstroud) [reply python-libraries/nanover-omni/src/nanover/omni/ase.py]: I've now moved these lines to the `make_regular_frame` function, let me know if what I've done seems alright to you!

**Review by Ragzouken — APPROVED** [review]
Looks good to me!

**Pull request merged**